portal/src/texts/profile/animation.mdx

```diff
@@ -0,0 +1,121 @@
+</Ingress>
+
+## Hvorfor?
+Vi bruker animasjon for å:
+* Skape særpreg
+* Gi en bedre brukeropplevelse
+* Hjelpe og rettlede brukeren
+* Skape fokus
+* Kamuflere nedlastingstid (underholde)
+
+Husk at animasjon alltid skal kunne slås av dersom en bruker ikke ønsker dette.
+Pass derfor på at navigasjon og interaksjon fungerer som det skal og er logisk uten animasjon
+
+## Grunnprinsipper
+### Overganger
+Animasjon bedrer din brukeropplevelse på en subtil måte og samtidig fremhever dine relevante oppgaver. 
+Ved hjelp av easingkurver og målrettet bruk av parallakseffekter, skaper vi smidige overganger som understøtter en god brukerreise.
+<DoDontExample
+    type="do"
+    description="Bruk easinger og ta hensyn til avstand."
+    content=<iframe src="https://player.vimeo.com/video/504726780?background=1&autoplay=1&loop=1" width="100%" height="250" frameborder="0" />
+/>
+<DoDontExample
+    type="dont"
+    description="Ikke bruk lineær easing."
+    content=<iframe src="https://player.vimeo.com/video/504726762?background=1&autoplay=1&loop=1" width="100%" height="250" frameborder="0" />
+/>
+
+### Posisjon
+Ved interaksjon, flyttes komponenter til en definert posisjon eller fikseres i en posisjon. 
+Dette hjelper brukeren til å lettere navigere på siden. 
+For å skape konsistens, fokuserer vi på horisontale og vertikale animasjoner.
+
+### Timing
+Varigheten av en animasjon avhenger av størrelsen på elementene, tilbakelagt avstand og forholdet til andre elementer. 
+Hvis det er flere elementer, bør timingen justeres slik at en klar sekvens opprettes i henhold til ønsket oppfatning. 
+Ved mange elementer, anbefales det kortere animasjoner.
+<DoDontExample
+    type="do"
+    description="Gjør bevegelser raskt nok til å unngå unødvendig lange ventetider."
+    content=<iframe src="https://player.vimeo.com/video/505647776?background=1&autoplay=1&loop=1" width="100%" height="250" frameborder="0" />
+/>
+<DoDontExample
+    type="dont"
+    description="Ikke forsink bevegelsene så mye at du må vente unødvendig."
+    content=<iframe src="https://player.vimeo.com/video/505647739?background=1&autoplay=1&loop=1" width="100%" height="250" frameborder="0" />
+/>
+
+### Gjennomsiktighet
+Gjennomsiktighet brukes som et verktøy for å markere elementer på en side på en jevn og behagelig måte. 
+Gjennomsiktighet kan kombineres med andre animasjonsparametere, som bevegelse, for en mer dynamisk opplevelse. 
+Oppfatningen av et element endres over en kort eller lang avstand.
+
+### Oppmerksomhet
+Animasjon tiltrekker oppmerksomhet. Husk at siste element som beveger seg er det som tiltrekker oppmerksomhet. 
+Bruk dette for sette fokus der du ønsker.
+
+Den visuelle profilen til Fremtind har et minimalistisk uttrykk, basert på enkle former og nøktern bruk av farger. 
+For å unngå å oppfattes som kalde og harde, bruker vi animasjon som et virkemiddel for å gjøre løsningene mer dynamisk og levende,
+i tilleg til å gi dem personlighet og stil. 
```

Comment:
Fint formulert. Tenker dette avsnittet hører hjemme på toppen av siden, under "hvorfor" :)

Reply:
Liten skriveleif: tillegg med to g.